src/redux/music/albumSlice.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable */
```

Comment:
- [ ] It will be great to fix the eslint errors. Kindly don't disable eslint in  your files.